src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -229,6 +230,20 @@ public virtual int MaximumRedirection
         }
         private int _maximumRedirection = -1;
 
+        /// <summary>
+        /// gets or sets the RetryCount property
```

Comment:
Please add a more meaningful summary. Capitalize the first work and end with a period.

Reply:
Fixed.